specification/templates/CONTRIBUTING.md

```diff
@@ -75,6 +75,6 @@ any 'help wanted' issues is a great place to start.
 See the [LICENSE](LICENSE) file for our repository's licensing. We will ask you to
 confirm the licensing of your contribution.
 
-We may ask you to sign a [Contributor License Agreement
-(CLA)](http://en.wikipedia.org/wiki/Contributor_License_Agreement) for larger
-changes.
+## CLA
```

Comment:
Is a new section needed? The contributor license agreement seems appropriate to be included in the `Licensing` section itself.

```suggestion
```

If not, can we make it a sub-section of `Licensing` and not use an acronym here?

```suggestion
### Contributor License Agreement
```

Reply:
Good point, made it a subsection.